scripts/release.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable camelcase */
+/* eslint-disable no-console */
+import fetch from 'node-fetch';
+
+const version = process.env.VERSION;
+
+const createTag = async () => {
+  try {
+    const payload = {
+      tag_name: `${process.env.PACKAGE}-v${version}`,
```

Comment:
Does this create a tag?

Reply:
Yes the request creates a tag and a release. I initially had some code to create a tag through the shell but this actually does both. Also found out that we can actually prefix the tags.